src/applications/appeals/10182/components/NeedsToVerify.jsx

```diff
@@ -3,6 +3,8 @@ import PropTypes from 'prop-types';
 
 const NeedsToVerify = ({ pathname }) => (
   <va-alert status="warning">
+    {/* <a href="tel:888-442-4551">888-442-4551 (711)</a> */}
+    <va-telephone contact="8884424551" />
```

Comment:
I'm not sure why this phone number is being added above the header? I don't think it needs it.

Reply:
Oh oops, I didn't realize this was closed LOL